packages/esbuild-plugin/jest.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  testEnvironment: "node",
+  transform: {
+    "^.+\\.(t|j)sx?$": ["@swc/jest"],
+  },
+};
```

Comment:
Totally optional:
Any change, we can also extract this to a central package?
We might wanna add a "dev-utils" package (I know this sounds oddly familiar....)

Reply:
If we notice that having stuff duplicated becomes a problem we can extract this. I usually don't like drying things too early. Same goes for other comments.